sync/src/relayer/compact_block_process.rs

```diff
@@ -226,8 +226,6 @@ impl<'a> CompactBlockProcess<'a> {
                 }
             }
 
-            assert!(!missing_transactions.is_empty() || !missing_uncles.is_empty());
```

Comment:
If there are no missing transactions and missing uncles, it should return `Ok(Status::AcceptBlock);` https://github.com/nervosnetwork/ckb/pull/1680/files#diff-98eccb89a5611099d6adca819c9c8b08L202

Reply:
```

Reply:
If this is a collision, the `missing_transactions` is not empty.